psygnal/_signal.py

```diff
@@ -912,3 +919,40 @@ def _is_subclass(left: AnyType, right: type) -> bool:
         if origin is Union:
             return any(issubclass(i, right) for i in get_args(left))
     return issubclass(left, right)
+
+
+def partial_weakref(partial_fun: partial) -> Callable:
+    obj, name = _get_proper_name(partial_fun.func)
+    args_ = partial_fun.args
+    kwargs_ = partial_fun.keywords
+
+    def wrap(*args, **kwargs):  # type: ignore
```

Comment:
I have no idea if it could be annotated in any way.

Reply:
```suggestion
    def wrap(*args: Any, **kwargs: Any) -> Any:
```

works for me